config.go

```diff
@@ -197,9 +197,9 @@ func WithTimer[H Hash](t Timer) func(config *Config[H]) {
 }
 
 // WithSecondsPerBlock sets SecondsPerBlock.
-func WithSecondsPerBlock[H Hash](d time.Duration) func(config *Config[H]) {
+func WithSecondsPerBlock[H Hash](f func() time.Duration) func(config *Config[H]) {
```

Comment:
`WithTimePerBlock` then? We're changing the signature anyway and `Seconds` are misleading.

Reply:
> WithTimePerBlock then?

Last time (with update on `time.Duration`) it wasn't changed, so this time I kept it as is. Will fix.